test/support/resources/comment.ex

```diff
@@ -35,6 +35,22 @@ defmodule AshGraphql.Test.Comment do
     read :paginated do
       pagination(required?: true, offset?: true, countable: true)
     end
+
+    action :ranked_comments, {:array, RankedComment} do
+      run(fn _input, _ctx ->
+        res =
+          Ash.read!(__MODULE__)
```

Comment:
I didn't get as far as seeing if I had to do some fixturing before this will actually return something, since it won't compile, so I have no idea if this will work or not. But I think the whole test is a feasible and nice thing to have tested.